src/main/java/com/hpw/messages/ErrorMessages.java

```diff
@@ -2,6 +2,8 @@
 
 public class ErrorMessages {
 
+
```

Comment:
why there are just white spaces?? I can not see any changes rather than that? What is was the purpose here?